apps/wallet-mobile/src/WalletInit/WalletForm.tsx

```diff
@@ -118,6 +113,10 @@ export const WalletForm = ({onSubmit}: Props) => {
           testID="walletFormContinueButton"
         />
       </Actions>
+
+      {Platform.OS === 'ios' && <KeyboardSpacer />}
+
+      {Platform.OS === 'android' && <Spacer height={16} />}
```

Comment:
the same thing happening here

https://github.com/Emurgo/yoroi/pull/2856/files#r1392753249